themes/sutro/template.html

```diff
@@ -0,0 +1,692 @@
+<!-- Sutro -->
+<style>
+  media-controller {
+    --base: 16px;
+    min-width: 300px;
+
+    font-size: calc(0.75 * var(--base));
+    font-family: Roboto, Arial, sans-serif;
+    --media-font-family: Roboto, helvetica neue, segoe ui, arial, sans-serif;
+    -webkit-font-smoothing: antialiased;
+
+    --media-primary-color: #fff;
+    --media-secondary-color: transparent;
+    --media-menu-background: rgba(28, 28, 28, 0.9);
+    --media-control-hover-background: var(--media-secondary-color);
+
+    --media-range-track-height: calc(0.125 * var(--base));
+    --media-range-thumb-height: var(--base);
+    --media-range-thumb-width: var(--base);
+    --media-range-thumb-border-radius: var(--base);
+
+    --media-preview-thumbnail-border: calc(0.125 * var(--base)) solid #fff;
+    --media-preview-thumbnail-border-radius: calc(0.125 * var(--base));
+    --media-preview-time-margin: calc(0.25 * var(--base)) 0 0;
+  }
+
+  media-controller[breakpointmd] {
+    --base: 20px;
+  }
+
+  /* The biggest size controller is tied to going fullscreen
+      instead of a player width */
+  media-controller[mediaisfullscreen] {
+    --base: 28px;
+  }
+
+  .media-button {
+    position: relative;
+    display: inline-block;
+    width: calc(2 * var(--base));
+    padding: 0;
+    height: 100%;
+    opacity: 0.9;
+    transition: opacity 0.1s cubic-bezier(0.4, 0, 1, 1);
+  }
+
+  .media-button svg {
+    height: 100%;
+    width: 100%;
+    fill: none;
+    stroke: #fff;
+    stroke-width: 1;
+    stroke-linecap: 'round';
+    stroke-linejoin: 'round';
+  }
+
+  svg .svg-shadow {
+    stroke: #000;
+    stroke-opacity: 0.15;
+    stroke-width: 2px;
+    fill: none;
+  }
+</style>
+
+<media-controller
+  breakpoints="md:480"
+  defaultsubtitles="{{defaultsubtitles}}"
+  defaultduration="{{defaultduration}}"
+  gesturesdisabled="{{disabled}}"
+  hotkeys="{{hotkeys}}"
+  nohotkeys="{{nohotkeys}}"
+  defaultstreamtype="on-demand"
+>
+  <slot name="media" slot="media"></slot>
+  <slot name="poster" slot="poster"></slot>
+
+  <!-- Gradient -->
+  <style>
+    .media-gradient-bottom {
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      height: calc(7 * var(--base));
+      background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6));
```

Comment:
it's a bit more code but easing gradients make a big difference on light backgrounds
https://github.com/muxinc/elements/pull/885